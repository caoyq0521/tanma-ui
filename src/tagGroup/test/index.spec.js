import { mount } from '@vue/test-utils';
import TmTagGroup from '../tagGroup.vue';

describe('tmTagGroup', () => {
  it('render tag-group', () => {
    const wrapper = mount(TmTagGroup);
    expect(wrapper).toMatchSnapshot();
  });
})
  