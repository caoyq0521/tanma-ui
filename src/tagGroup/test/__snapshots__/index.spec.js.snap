// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tmTagGroup render tag-group 1`] = `<span class="tm-tag-more"><div role="tooltip" id="el-popover-8637" aria-hidden="true" class="el-popover el-popper" style="width: 292px; display: none;" name="fade-in-linear" tabindex="0"><!----></div><div class="reference el-popover__reference" aria-describedby="el-popover-8637" tabindex="0"><i class="tm-icon-gengduo2 tm-tag-more__icon"></i></div></span>`;
